### ISSUE TITLE
Create index map legend to indicate what each color means

### ISSUE THREAD

**Issue by mferrarini** [geoblacklight/geoblacklight#1156]
### Descriptive summary

HGL has done and analysis and integration of color combinations based on colorblindness. Create an legend to appear above the index map indicating what each color means

### Related work
* Issue #1137

### Screenshot example
![Screen Shot 2022-02-16 at 1 44 25 PM](https://user-images.githubusercontent.com/13121501/154334290-8ede6f94-f2fc-46d5-a91d-79d4cc497e96.png)

**Comment by kgjenkins** [comment]
Thanks for investigating this, @mferrarini -- this is definitely an improvement over the old green/red color scheme!

On the map, there is some level of transparency happening (not sure what %) so that the colors get washed out a bit, and make the "Map held" and "Selected" colors a bit closer.  Is there a way to test the accessibility of these colors that would account for that transparency level?

Also, it would be great to have the legend colors exactly match the map colors, so maybe we could apply the same level of transparency to the legend colors.

**Comment by mferrarini** [comment]
Updated legend colors (screenshot of local environment)
![Screen Shot 2022-02-22 at 10 48 43 AM](https://user-images.githubusercontent.com/13121501/155168351-d57c6f86-e7be-4759-8843-8a436ca6d1c4.png)
